test/unit/lib/plugins/aws/package/compile/functions.test.js

```diff
@@ -1875,12 +1885,17 @@ describe('lib/plugins/aws/package/compile/functions/index.test.js', () => {
               },
             },
             fnImage: {
-              image:
-                '000000000000.dkr.ecr.sa-east-1.amazonaws.com/test-lambda-docker@sha256:6bb600b4d6e1d7cf521097177dd0c4e9ea373edb91984a505333be8ac9455d38',
+              image: imageWithSha,
             },
             fnImageWithTag: {
               image: '000000000000.dkr.ecr.sa-east-1.amazonaws.com/test-lambda-docker:stable',
             },
+            fnImageWithExplicitUri: {
```

Comment:
Let's maybe prefix name with `fnProviderImage` to differentiate those which have _image_ config inline and those which refer to `provider.images`